README.md

```diff
@@ -1,41 +1,82 @@
+    <a href="https://raw.githubusercontent.com/iotaledger/trinity-wallet/develop/LICENSE" style="text-decoration:none;"><img src="https://img.shields.io/badge/License-Apache%202.0-green.svg" alt="Apache 2.0 license"></a>
+    <a href="https://dependabot.com" style="text-decoration:none;"><img src="https://api.dependabot.com/badges/status?host=github&repo=iotaledger/trinity-wallet" alt=""></a>
+</p>
+      
+<p align="center">
+  <a href="#about">About</a> ◈
+  <a href="#prerequisites">Prerequisites</a> ◈
+  <a href="#installation">Installation</a> ◈
+  <a href="#getting-started">Getting started</a> ◈
+  <a href="#supporting-the-project">Supporting the project</a> ◈
+  <a href="#joining-the-discussion">Joining the discussion</a> 
+</p>
+
+---
+
+## About
+
+This is the **official** IOTA wallet, which allows you to do the following:
+* Create password-protected accounts to store and access your seeds
+* Send transactions
+* Generate addresses
+* Read your balance and transaction history
+
+This is beta software, so there may be performance and stability issues.
+Please report any issues in our [issue tracker](https://github.com/iotaledger/trinity/issues/new).
 
-[![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=iotaledger/trinity-wallet)](https://dependabot.com)
+## Prerequisites
 
+To compile Trinity, you must have the following:
 
-This is the shared repository for the IOTA Trinity Mobile and Desktop wallets. The apps are based on React and built with React-Native (mobile) and Electron (desktop).
+* Node.js 8 or higher. We recommend the [latest LTS](https://nodejs.org/en/download/)
+* [Yarn](https://yarnpkg.com/) package manager.
 
-## Prerequisites
+## Installation
 
-- Node.js (8+)
-- Yarn
+To install the dependencies needed to compile both Trinity Mobile and Trinity Desktop, do the following:
 
-## Instructions
+1. Clone this repository
 
-1. Clone this repo
-```
-git clone https://github.com/iotaledger/trinity-wallet.git
-```
+    ```
+    git clone https://github.com/iotaledger/trinity-wallet.git
+    ```
 
-2. Go to the trinity-wallet directory
-```
-cd trinity-wallet
-```
+2. Change into the `trinity-wallet` directory
+
+    ```
+    cd trinity-wallet
+    ```
 
 3. Install the shared dependencies
-```
-yarn && yarn deps:shared
-```
 
-4. Follow the instructions to install dependencies for [Trinity Mobile](https://github.com/iotaledger/trinity-wallet/blob/develop/src/mobile/README.md) or [Trinity Desktop](https://github.com/iotaledger/trinity-wallet/blob/develop/src/desktop/README.md)
+    ```
+    yarn && yarn deps:shared
+    ```
+
+## Getting started
 
+To compile Trinity, follow the instructions for either [Trinity Mobile](https://github.com/iotaledger/trinity-wallet/blob/develop/src/mobile/README.md) or [Trinity Desktop](https://github.com/iotaledger/trinity-wallet/blob/develop/src/desktop/README.md).
```

Comment:
I think it might be better to have this in the "Installation" section, as the shared dependencies should be installed before installing the dependencies specific to mobile or desktop

Reply:
Added this to installation 👍